test/unit/services/platform-test.js

```diff
@@ -74,4 +75,27 @@ describe('Platform', function() {
 
   });
 
+  describe('isVirtualizationEnabled', function(){
+
+    it('should able to check for mac', function() {
+      let stub = sandbox.stub(Platform, 'getOS').returns('darwin');
+      Platform.isVirtualizationEnabled();
+      expect(stub).to.have.been.calledOnce;
+    });
+
+    it('should able to check for linux', function() {
+      let stub = sandbox.stub(Platform, 'getOS').returns('linux');
+      Platform.isVirtualizationEnabled();
+      expect(stub).to.have.been.calledOnce;
+    });
+
+    it('should able to check for windows', function() {
+      let stub = sandbox.stub(Platform, 'getOS').returns('win32');
+      let stubs = sandbox.stub(child_process, 'spawn').yields(undefined, 'stdout', 'stderr');
+      Platform.isVirtualizationEnabled();
+      expect(stub).to.have.been.calledOnce;
```

Comment:
again, there is no need to test that Platform.getOS stub is called, that is responsibility for platform-test.js.